arm9/source/system/vram_data.s

```diff
@@ -0,0 +1,12 @@
+.section .rodata.vram_data
+
+.align 2
+.global vram_data
+vram_data:
+	.incbin "../output/vram0.tar"

```

Comment:
I'm not thrilled with the hardcoded path, or even worse making a reference to something a directory up, but it works and needed minimal changes to implement.